data/mods/gen5/formats-data.ts

```diff
@@ -2659,7 +2659,7 @@ export const BattleFormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 		tier: "LC",
 	},
 	beartic: {
-		randomBattleMoves: ["aquajet", "iciclecrash", "stoneedge", "superpower", "taunt"],
+		randomBattleMoves: ["aquajet", "encore", "iciclecrash", "stoneedge", "superpower"],
```

Comment:
Encore? Why is that better than Taunt?

Reply:
slow encore is better than slow taunt

Reply:
and also everyone in set workshop absolutely despised taunt